themes/powerline/powerline.base.bash

```diff
@@ -255,6 +255,7 @@ function __powerline_prompt_command() {
 	LEFT_PROMPT=""
 	SEGMENTS_AT_LEFT=0
 	LAST_SEGMENT_COLOR=""
+	save-and-reload-history "${HISTORY_AUTOSAVE:-0}"
```

Comment:
/nitpick For consistency, let's add a blank line above this to separate it from the assignment block